.github/workflows/provider-updoc.yml

```
@@ -2,13 +2,19 @@ name: Updoc
 
 on:
   workflow_call:
+    inputs:
+      publish-pack:
```

nit: We may use a more explanatory name for this parameter:
```suggestion
      providers:
```

---

Done.